### DISCUSSION TITLE
How can I remove logo text ?

### DISCUSSION THREAD

**Discussion by mahmutoz** [vuejs/vitepress#2256] [Q&A] [answered]
I want to remove "test" text in logo. 

<img width="111" alt="Screenshot 2023-04-18 at 15 23 30" src="https://user-images.githubusercontent.com/49031200/232777580-8249be06-3f96-413a-b202-83e12ad08e62.png">

```html
<a class="title" href="/">
  <img class="VPImage dark logo" src="/logo/logo-light.svg" alt="Logo">
  test
</a>
```

I removed title but came "VitePress" text as the default

```js
export default defineConfig({
  title: "test"
})
```

How can I solve this?

**Comment by brc-dd** [comment, marked as answer]
[`siteTitle: false`](https://vitepress.dev/reference/default-theme-nav#site-title-and-logo)

**Comment by mahmutoz** [comment]
Thank you.
